spec/integration/tagging_things_spec.rb

```diff
@@ -0,0 +1,179 @@
+require "rails_helper"
```

Comment:
Before merging this PR we should probably remove this file. It's a good thing to have during this migration, but it's (hopefully) not testing anything new.